x/crosschain/types/inbound_params.go

```diff
@@ -2,19 +2,13 @@ package types
 
 import (
 	"fmt"
-
-	"github.com/zeta-chain/zetacore/pkg/chains"
 )
 
 func (m InboundParams) Validate() error {
 	if m.Sender == "" {
 		return fmt.Errorf("sender cannot be empty")
 	}
 
-	if chains.GetChainFromChainID(m.SenderChainId) == nil {
```

Comment:
We're removing here and other places in `types` like packages checks for chain ID validity.

The reasoning is that chain ID validity check is no longer stateless as chain IDs can be added dynamically as part of the chain state